src/services/verifier.rs

```diff
@@ -1193,4 +1196,16 @@ mod tests {
         );
         res
     }
+
+    #[test]
+    fn format_attribute() {
+        assert_eq!(normalize_encoded_attr(""), "");
+        assert_eq!(normalize_encoded_attr("abc"), "abc");
+        assert_eq!(normalize_encoded_attr("0"), "0");
+        assert_eq!(normalize_encoded_attr("01"), "1");
+        assert_eq!(normalize_encoded_attr("01.0"), "01.0");
+        assert_eq!(normalize_encoded_attr("0abc"), "0abc");
+        assert_eq!(normalize_encoded_attr("-100"), "-100");
+        assert_eq!(normalize_encoded_attr("-0100"), "-100");
```

Comment:
Would you expect this to have the leading 0 trimmed?

We need to make sure the spec aligns with this behavior

Reply:
The spec mentions  `convert any string integer (e.g. "1234") to be a 32-bit integer (e.g. 1234)`. Since integers can be negative, I thought this was the correct way. Maybe @swcurran can offer some clarification on this?

Reply:
That makes sense yes 👍